client/components/Header.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+const Header = () => {
+  return (
+    <div className='header'>
+      <h1>Pings</h1>
+    </div>
+  )
+}
```

Comment:
You could add a buttons/links (Link tag) that redirects to the /new page and the /profile page (and back to '/' again)